src/_pytest/config/__init__.py

```diff
@@ -341,6 +343,32 @@ def _get_directory(path: Path) -> Path:
         return path
 
 
+def _get_legacy_hook_marks(
```

Comment:
This will return the incorrect value (and emit an erroneous error message) if someone sets an option to `False`:

```python
pytest_runtest_call.tryfirst = False
```

While this is unusual, I can definitely see this happening if someone had that set to `True`, and for some reason decided to then set to `False`.